var/spack/repos/builtin/packages/subread/package.py

```diff
@@ -43,6 +43,11 @@ def build(self, spec, prefix):
         plat = sys.platform
         with working_dir('src'):
             if plat.startswith('linux'):
+                filter_file(
+                    'CC_EXEC = gcc',
+                    'CC_EXEC = {0}'.format(self.compiler.cc),
```

Comment:
`self.compiler.cc` is the actual compiler, if you want to use the compiler wrappers you want `spack_cc`. `'cc'` would also work but relies on the `PATH` being set correctly, which has been buggy of late.